.travis.yml

```diff
@@ -77,10 +80,14 @@ matrix:
            CONDA_DEPENDENCIES=${CONDA_ALL_DEPENDENCIES}
            EVENT_TYPE='cron'
 
+
 install:
+  - sudo apt-get install libssl1.0.0
```

Comment:
Why do we need to install `libssl` and why do we use `sudo` here?

Avoiding use of `sudo` is desirable so that we can run on the Travis container-based infrastructure. For `apt` you can use [Installing Packages on Container Based Infrastructure](https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Container-Based-Infrastructure) *without requiring `sudo`*.